phpBB/styles/prosilver/template/bbcode.html

```diff
@@ -11,6 +11,39 @@
 <!-- BEGIN quote_username_open --><blockquote><div><cite>{USERNAME} {L_WROTE}{L_COLON}</cite><!-- END quote_username_open -->
 <!-- BEGIN quote_open --><blockquote class="uncited"><div><!-- END quote_open -->
 <!-- BEGIN quote_close --></div></blockquote><!-- END quote_close -->
+<!-- BEGIN quote_extended -->
+<blockquote>
+	<xsl:if test="not(@author)">
+		<xsl:attribute name="class">uncited</xsl:attribute>
```

Comment:
Extra:
If a post is cited, you may get the username from the post_id.

Please check if that's viable. Otherwise, you may leave it empty